### ISSUE TITLE
Make Documentation Links that Show Up in Error Messages, Hyperlinks

### ISSUE THREAD

**Issue by sbudhirajadoc** [forcedotcom/salesforcedx-vscode#4640]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I see a link I cannot link on. It's even more frustrating when the link is in an error message. 

**Describe the solution you'd like**
I'd like any link that shows up in an error message to be clickable.

**Describe alternatives you've considered**
Copy paste the link into a browser tab but I think it is not possible to select text in a VS Code error message.

Error message screenshot: 
![Screen Shot 2023-01-18 at 10 47 06 AM](https://user-images.githubusercontent.com/57969266/214121473-cf00cff8-f142-41cf-ae7f-ba6407d042fa.png)

**Additional context**

**Comment by CristiCanizales** [comment]
Hello! 🙂 
Regarding the alternative, it is possible to select text in a VS Code error message, so with "⌘ + C" the link could be taken from there. 
![image](https://user-images.githubusercontent.com/113132642/215111621-c62db5db-a8ad-4b0c-89fc-ed3f06a92d04.png)

Also, you can always click the link from the Output Tab, that one is easily clickable (⌘ + click).
![image](https://user-images.githubusercontent.com/113132642/215114211-ab462090-1f56-42db-975f-08366f052de0.png)

Nonetheless, I agree it would definitely be better to be able to just click it from the notification.

**Comment by sbudhirajadoc** [comment]
Agreed. it is not a good user experience to see an un-clickable link or to have to copy-paste to view the doc.
